Ports/zsh/package.sh

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=zsh
-version=5.8.1
-files="https://sourceforge.net/projects/zsh/files/zsh/${version}/zsh-${version}.tar.xz zsh-${version}.tar.xz b6973520bace600b4779200269b1e5d79e5f505ac4952058c11ad5bbf0dd9919"
+version=5.9
+files="https://sourceforge.net/projects/zsh/files/zsh/${version}/zsh-${version}.tar.xz zsh-${version}.tar.xz 9b8d1ecedd5b5e81fbf1918e876752a7dd948e05c1a0dba10ab863842d45acd5"
 auth_type="sha256"
 useconfigure=true
 use_fresh_config_sub=true
 
 pre_configure() {
     run "./Util/preconfig"
 }
+
+build() {
+    run_replace_in_file "s/define HAVE_PRCTL 1/undef HAVE_PRCTL/" config.h
+}
```

Comment:
If you override `build`, `make` will no longer run? Maybe you can use `post_configure()` for this instead?

Reply:
Well it does build and install. But sure I can switch it.

Reply:
Yeah, since you're saved by the fact that `make install` also builds the objects. But they're being built in the wrong step if you override `build()`.

Reply:
I took the time to also look into the code what functions we do support and add them to the README, there are now 3 new functions documentated.